jerry-core/ecma/builtin-objects/ecma-builtin-global.inc.h

```diff
@@ -139,6 +139,13 @@ OBJECT_VALUE (LIT_MAGIC_STRING_MATH_UL,
               ECMA_BUILTIN_ID_MATH,
               ECMA_PROPERTY_CONFIGURABLE_WRITABLE)
 #endif /* ENABLED (JERRY_BUILTIN_MATH) */
+#if ENABLED (JERRY_ES2015_BUILTIN) && ENABLED (JERRY_BUILTIN_REFLECT)
```

Comment:
`ENABLED (JERRY_BUILTIN_REFLECT)` involves the `ENABLED (JERRY_ES2015_BUILTIN)` guard.